docs/index.rst

```diff
@@ -113,73 +122,12 @@ nature, you get a fast DataFrame at 1MB and 1TB+.
 
 **Modin is currently under active development. Requests and contributions are welcome!**
 
-**If you are interested in contributions please refer to 'developer documentation' section, where
```

Comment:
I feel it a little confusing when I open the main index page, and there's nothing on the left to navigate - I only have a "this page contains" navbar on the right, and I have to notice the upper navbar which is something I'm not used to.

Can we duplicate the upper menu here on the left then?

Reply:
That's a good observation. The way pandas readthedocs gets around this is by having big icons for its four navbar sections. I think this would be better added in a following pull request, perhaps implemented similarly to how pandas does it.